pkgs/development/python-modules/roombapy/default.nix

```
@@ -0,0 +1,20 @@
+{ buildPythonPackage, fetchPypi, lib, paho-mqtt }:
+
+buildPythonPackage rec {
+  pname = "roombapy";
+  version = "1.6.2.post1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0rw60j6ygfbl4mplpgz6iiambv2gbdlf0wgfp4z3z52a2sm0n9a8";
+  };
+
+  propagatedBuildInputs = [ paho-mqtt ];
+
```

```suggestion
  pythonImportsCheck = [ "roombapy" ];
```

see https://nixos.org/manual/nixpkgs/unstable/#python

---

Since unit tests are now being run, does that make `pythonImportsCheck` irrelevant?

---

Not completely. It is not a problem when we have both because it makes sure the package is importable. Ideally every package should have it.

---

Makes sense, added.